### DISCUSSION TITLE
Email wallet approve transaction modal with no content

### DISCUSSION THREAD

**Discussion by jplsaez** [WalletConnect/walletconnect-monorepo#4498] [Developer Support] [answered]
Hello.

I managed to successfully signing in through the new email wallet feature and connecting to my testing DApp.

When I try to issue a write action, the approve transaction modal is shown but no content inside. Please see attached screenshot.

On the other hand, the createWeb3Modal object's open() method successfully opens the wallet modal, as the attached screenshot shown.

Any hints about this issue?

Best regards.

   Juan Pedro López

![open_wallet](https://github.com/WalletConnect/walletconnect-monorepo/assets/113118971/489a5729-50f1-44d1-b10b-a1b24889e031)
![approve_transaction](https://github.com/WalletConnect/walletconnect-monorepo/assets/113118971/951c8e55-9a7b-41ba-a603-90a65e4be7b2)

**Comment by glitch-txs** [comment, marked as answer]
hi, could you open an issue in the Web3Modal repo? also please share a repro example and steps to reproduce 👍 thanks!

**Comment by jplsaez** [comment]
Hi,

While preparing a test case, I saw a new version of ***@***.***/wagmi"
named "alpha" has been released.

This way, the problem has disappeared so you can close the issue.

Best regards.

   Juan Pedro López

**Reply by wallneradam** [reply]
I have the same problem. I don't see a new version anywhere, but an older one. There is a newer (canary) version, which has the same bug.

**Reply by wallneradam** [reply]
But for me the 4.2.0-alpha.0 do the same.
